modules/account-lib/test/unit/coin/dot/transactionBuilder/withdrawUnstakedBuilder.ts

```diff
@@ -0,0 +1,121 @@
+        .sender({ address: sender.address })
+        .validity({ firstValid: 3933, maxDuration: 64 })
+        .referenceBlock(refBlock)
+        .sequenceId({ name: 'Nonce', keyword: 'nonce', value: 200 })
+        .fee({ amount: 0, type: 'tip' });
+      builder.sign({ key: sender.secretKey });
+      const tx = await builder.build();
+      const txJson = tx.toJson();
+      should.deepEqual(txJson.numSlashingSpans, '0');
+      should.deepEqual(txJson.sender, sender.address);
+      should.deepEqual(txJson.blockNumber, 3933);
+      should.deepEqual(txJson.referenceBlock, refBlock);
+      should.deepEqual(txJson.genesisHash, genesisHash);
+      should.deepEqual(txJson.specVersion, specVersion);
+      should.deepEqual(txJson.nonce, 200);
+      should.deepEqual(txJson.tip, 0);
+      should.deepEqual(txJson.transactionVersion, txVersion);
+      should.deepEqual(txJson.chainName, chainName);
+      should.deepEqual(txJson.eraPeriod, 64);
+    });
+
+    it('should build an unsigned withdrawUnstaked transaction', async () => {
+      builder
+        .slashingSpans(0)
+        .sender({ address: sender.address })
+        .validity({ firstValid: 3933, maxDuration: 64 })
+        .referenceBlock(refBlock)
+        .sequenceId({ name: 'Nonce', keyword: 'nonce', value: 200 })
+        .fee({ amount: 0, type: 'tip' });
+      const tx = await builder.build();
+      const txJson = tx.toJson();
+      should.deepEqual(txJson.numSlashingSpans, '0');
+      should.deepEqual(txJson.sender, sender.address);
+      should.deepEqual(txJson.blockNumber, 3933);
+      should.deepEqual(txJson.referenceBlock, refBlock);
+      should.deepEqual(txJson.genesisHash, genesisHash);
+      should.deepEqual(txJson.specVersion, specVersion);
+      should.deepEqual(txJson.nonce, 200);
+      should.deepEqual(txJson.tip, 0);
+      should.deepEqual(txJson.transactionVersion, txVersion);
+      should.deepEqual(txJson.chainName, chainName);
+      should.deepEqual(txJson.eraPeriod, 64);
+    });
+
+    it('should build from raw signed tx', async () => {
+      builder.from(rawTx.withdrawUnbonded.signed);
+      builder.validity({ firstValid: 3933 }).referenceBlock(refBlock);
+      const tx = await builder.build();
+      const txJson = tx.toJson();
+      should.deepEqual(txJson.numSlashingSpans, '0');
+      should.deepEqual(txJson.sender, sender.address);
+      should.deepEqual(txJson.blockNumber, 3933);
+      should.deepEqual(txJson.referenceBlock, refBlock);
+      should.deepEqual(txJson.genesisHash, genesisHash);
+      should.deepEqual(txJson.specVersion, specVersion);
+      should.deepEqual(txJson.nonce, 200);
+      should.deepEqual(txJson.tip, 0);
+      should.deepEqual(txJson.transactionVersion, txVersion);
+      should.deepEqual(txJson.chainName, chainName);
+      should.deepEqual(txJson.eraPeriod, 64);
+    });
+
+    it('should build from raw unsigned tx', async () => {
+      builder.from(rawTx.withdrawUnbonded.signed);
```

Comment:
```suggestion
      builder.from(rawTx.withdrawUnbonded.unsigned);
```

I think you meant to use the unsigned txn here

Reply:
Yes good catch. Fixed.